### PR TITLE
Fix `order`s `newline-between` for multiline imports

### LINKED ISSUE #313: `order`'s `newlines-between` doesn't work with multiline import statements
Settings `newlines-between: always`

Not working examples:

```js
import FurtherStepsHeadline // invalid error report here: There should be no empty line within import group  import/order
   from '../../../../../../app/components/pages/contentFunnel/donePartials/FurtherStepsHeadline';
```

```js
import { expect } from 'chai';

import RangeType, { // invalid error report here: There should be no empty line within import group  import/order
    isFullRange,
    isPartialRange,
} from '../../../../app/lib/dateRange';
```

```js
local: {
	launch_url: 'http://localhost:${APPLICATION_PORT}',
	selenium_host: 'localhost',
	selenium_port: 4444,
	selenium: {
		start_process: true,
		server_path: require('selenium-server-standalone-jar').path, // invalid error report here: There should be no empty line within import group  import/order
		cli_args: {
			'webdriver.chrome.driver': require('chromedriver').path
		}
	},
	desiredCapabilities: {
		browserName: 'chrome'
	}
},
```

I will fix this ASAP.

### PR DESCRIPTION
**Pull request by singles**
Partly-fixes #313.

### CODE REVIEW

**Comment by singles** [comment]
See #313

File: tests/src/rules/order.js [hunk 1]
```diff
@@ -206,6 +206,70 @@ ruleTester.run('order', rule, {
...
+          from '../../../../this/will/be/very/long/path/and/therefore/this/import/has/to/be/in/two/lines';
+        import external from 'external'
+      `,
+      options: [
+        {
+          groups: [
+            ['builtin', 'index'],
+            ['sibling'],
+            ['parent', 'external'],
+          ],
```
Comment (jfmengels) [inline tests/src/rules/order.js]: Not that it matters all that much, but you could simplify the tests by removing the `groups` option, as you don't need it.

**Comment by jfmengels** [comment]
LGTM.

You say this partially fixes #313, what part of it remains? What I suggested adding tests for in https://github.com/benmosher/eslint-plugin-import/issues/313#issuecomment-217466177?

**Comment by singles** [comment]
@jfmengels Yes, this is pretty tricky, but looks like I'm not far away from finishing that. Also case with `require` in nightwatch config isn't solved yet.

File: tests/src/rules/order.js (hunk 1 above)
Comment (singles) [inline tests/src/rules/order.js]: Done.

Comment (jfmengels) [inline tests/src/rules/order.js]: I meant in all the tests you added and where it's not needed. I think you only removed it from the last one?

**Comment by jfmengels** [comment]
Not sure what you meant with `nightwatch config`..

**Comment by singles** [comment]
@jfmengels That part with requires of `selenium` and `chromedrvier` - it comes from our `nightwatch.js` file. Sorry for not being clear enough.

**Comment by singles** [comment]
I will create new PR which solves all issues mentioned in #313, closing that one.

**Pull request closed without merging**